static/styles/portico/portico-signin.scss

```diff
@@ -1,3 +1,6 @@
+// The width of all input fields in registration form
+$input-width: 350px;
```

Comment:
I think it would be better to let the form have an inner width of 350px and let the inputs have 100% width. We shouldn't needlessly complicate our CSS with variables.

Reply:
Yeah, that seems right to me.